Sources/Plasma/FeatureLib/pfPython/pyVaultMarkerGameNodeGlue.cpp

```diff
@@ -130,17 +130,13 @@ PYTHON_METHOD_DEFINITION(ptVaultMarkerGameNode, setMarkers, args)
         PyObject* age  = PySequence_GetItem(marker_seq, 1);
         PyObject* pos  = PySequence_GetItem(marker_seq, 2);
         PyObject* desc = PySequence_GetItem(marker_seq, 3);
-        if (!(PyInt_Check(id) && PyString_CheckEx(age) && pyPoint3::Check(pos) && PyString_CheckEx(desc))) {
+        if (!(PyLong_Check(id) && PyUnicode_Check(age) && pyPoint3::Check(pos) && PyUnicode_Check(desc))) {
             PyErr_SetString(PyExc_TypeError, errmsg);
             PYTHON_RETURN_ERROR;
         }
 
-        VaultMarker marker;
-        marker.id = PyLong_AsUnsignedLong(id);
-        marker.age = PyString_AsStringEx(age);
-        marker.pos = pyPoint3::ConvertFrom(pos)->fPoint;
-        marker.description = PyString_AsStringEx(desc);
-        collector.push_back(marker);
+        collector.push_back({ PyLong_AsUnsignedLong(id), PyUnicode_AsSTString(age),
+                              pyPoint3::ConvertFrom(pos)->fPoint, PyUnicode_AsSTString(desc) });
```

Comment:
`emplace_back`?

Reply:
I tried that, and it resulted in a compile error :sad:

Reply:
... It helps if `VaultMarkers` has a constructor, doesn't it? Sigh.

Reply:
Hmm, it looks like you can do it as `vector.emplace_back(StructType { ... })`...  However, a quick test shows that that version isn't any more optimal than the `push_back` with optimization turned on :(.

It looks like this will get fixed in C++20, but for now I think it's fine to leave the `push_back`.

Reply:
Or just add a constructor like you did -- that works too :)